RELEASE-NOTES.txt

```diff
@@ -1,5 +1,6 @@
 Unreleased
 ------
+* [**] Fix incorrect block insertion point after blurring the post title field. [https://github.com/WordPress/gutenberg/pull/32831]
```

Comment:
I see we usually reference the Gutenberg Mobile PR instead of the Gutenberg one so what do you think about updating this entry with the following suggestion?

```suggestion
* [**] Fix incorrect block insertion point after blurring the post title field. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3640]
```